### ISSUE TITLE
Allow `podman start` to take environment variables

### ISSUE THREAD

**Issue by dengelt** [containers/podman#24875]
### Feature request description

`podman start` should be able to pass environment variables to the container. My use case is to turn DEBUG on/off for development, without recreating the container.

### Suggest potential solution

Adding the same `--env, -e=env` flags as for [podman run](https://docs.podman.io/en/latest/markdown/podman-run.1.html#env-e-env) would solve the problem.

### Have you considered any alternatives?

Recreating the container is a workaround, but that is annoying and cumbersome when you just want to pass a debugging flag once and the container is initially created and run by systemd or some other service management system.

### Additional context

The [same issue exists in docker](https://github.com/moby/moby/issues/7561) since over 10 years, with overwhelming community demand but unwillingness from the developers to implement it.

**Comment by Luap99** [comment]
I don't think it makes sense to do that. From a design like that it would totally unclear would the env set on one start stay across a restart or not? 

One thing that might make sense is to extend `podman update` to allow adding/removing env vars. This way it would be quite clear that they persist.

**Comment by jankaluza** [comment]
I will look into this one once I have some time this week.

**Comment by jankaluza** [comment]
It took me more time than I expected to finish the remaining tasks in my current team, but I've just started checking this issue again :-).
